app/javascript/components/widgets/twitter/tweet_profile.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import twitterIcon from '@images/twitter-mojo.svg';
+import { fontSizes, colors, spacing } from '../../../lib/theme';
+
+const ProfileWrapper = styled.div`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: space-between;
+  width: 16vw;
+  flex-direction: row;
+  align-items: flex-end;
+  margin-bottom: 48px;
+`;
+
+const ProfileInfoWrapper = styled.div`
+  display: flex;
+  flex-wrap: no-wrap;
+  justify-content: space-between;
+  flex-direction: column;
+`;
+
+const ProfileHeader = styled.div`
+  color: ${colors.white}
+  font-size: ${fontSizes.large}
+  margin-bottom: ${spacing.s}
+`;
+
+const ProfileSub = styled.div`
+  color: ${colors.white}
+  opacity: 0.5
+  font-size: ${fontSizes.small}
+`;
+
+const TwitterProfile = ({ dateCreated }) => {
+  return (
+    <ProfileWrapper>
+      <div style={{ marginRight: '24px' }}>
+        <img src={twitterIcon} alt="twitter profile" />
+      </div>
+      <ProfileInfoWrapper>
+        <ProfileHeader>MojoTech</ProfileHeader>
+        <div style={{ display: 'flex', flexDirection: 'row' }}>
+          <ProfileSub style={{ marginRight: '30px' }}>@MojoTech</ProfileSub>
```

Comment:
Maybe a line like this to get this more like the designs?
`<ProfileSub> • </ProfileSub>`
![Screen Shot 2019-06-26 at 1 58 35 PM](https://user-images.githubusercontent.com/48894331/60203382-846fc580-981a-11e9-8281-84c8c5833b5e.png)